### ISSUE TITLE
no detection after training

### ISSUE THREAD

**Issue by hashirali2604** [thtrieu/darkflow#887]
Hey everyone! Kindly help me as soon as you guys got my issue... So my concern is that I'm working on a project to recognize 12 dentist instruments, I performed all the steps but at the end nothing detected, I am worrying why it happened 😢😢
I'm using tensorflow CPU and I ended up training my dataset at 1375 steps as I noticed that it was continuously moving around 3... Also I put all the annotation files in one folder and images in the other...
Before this I have done recognizing 1 object (mouth mirror) and killed training at 750 step and it was detected...
Please help me what do now?﻿

**Comment by ashleyjsands** [comment]
Hi hashirali2604,

My guess is that your loss is still too high and that you need to train it longer if you can. If the loss plateaued around 3, then you can't train the model any more. If it didn't plateau, then you should let the model train longer, for days or even weeks until it stops improving. If you can't wait that long, then you should train the model on the fastest GPU you can afford.

In the case you can't train it longer, my next guess is that you don't have enough data, or that your data/annotations are poor quality. You may need to increase your dataset size by an order or magntitude or two.

This suggestions are just based on guesses about your situation. Please post more information.

Regards,
Ashley
